src/main/java/agh/ics/oop/RectangularMap.java

```diff
@@ -0,0 +1,58 @@
+package agh.ics.oop;
+
+import java.util.ArrayList;
+
+public class RectangularMap implements IWorldMap {
+    public final int WIDTH;
```

Comment:
dużymi literami piszemy tylko stałe (`static final`), to akurat jest normalny atrybut tworzony per obiekt mapy ;)